routes/api/post.js

```diff
@@ -1,9 +1,42 @@
 const express = require('express');
 const router = express.Router();
+const isSignedIn = require('../../middleware/auth');
+const { createPost, getPostById, updatePost, deletePost } = require('../../controllers/post');
```

Comment:
@ranjit977 please check the controllers file. I don't see you added this file into your pull requests

Reply:
@sergust I have just pushed that file too.